src/k8s.rs

```diff
@@ -1,13 +1,47 @@
+use std::fmt;
```

Comment:
🚫 **[clippy]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> 
<pre><code>error: unused import: `std::fmt`
 --> src/k8s.rs:1:5
  |
1 | use std::fmt;
  |     ^^^^^^^^
  |
  = note: `-D unused-imports` implied by `-D warnings`
  = help: to override `-D warnings` add `#[allow(unused_imports)]`
</code></pre>
<!-- __reviewdog__:ChA5YzAwNWJkNjlkY2MxZmJmEgZjbGlwcHk= -->